paper/forestecology.Rmd

```diff
@@ -458,7 +459,7 @@ The resulting RMSE of `r model_rmse_cv` computed using cross-validation is `r if
 
 # Importance of spatial cross-validation
 
-`run_cv()` also accepts the `run_shuffle` argument in order to permute competitor species identity as described in Section \ref{permutation-test}. Figure \ref{fig:scbi-simulation} compares model performance for 49 permutations of competitor species and RMSE calculations, both with and without cross-validation. Without cross-validation, competitor species identity does matter as the observed RMSE was significantly lower than the permutation null distribution of RMSE. However, once we incorporate spatial cross-validation, this improvement disappears. These results suggest that in this 9 ha subplot of the SCBI plot, competitive interactions do not depend on the identity of the competitor, which is the opposite of what has been observed in other locations [@allen_permutation_2020 @uriarte_spatially_2004]. Furthermore, the larger cross-validated RMSE's are indicative of the importance of being vigilant against potential model overfitting.
+`run_cv()` also accepts the `run_shuffle` argument in order to permute competitor species identity as described in Section \ref{permutation-test}. Figure \ref{fig:scbi-simulation} compares model performance for 49 permutations of competitor species and RMSE calculations, both with and without cross-validation. Without cross-validation, competitor species identity does matter as the observed RMSE was significantly lower than the permutation null distribution of RMSE. However, once we incorporate spatial cross-validation, this improvement disappears. These results suggest that in this 9 ha subplot of the SCBI plot, competitive interactions do not depend on the identity of the competitor, which is the opposite of what has been observed in other locations [@allen_permutation_2020 @uriarte_spatially_2004]. This provides a striking example of the importance of cross-validation, as without it the over-fit model gives rise to an incorrect conclusion.
```

Comment:
HEre I just really wanted to emphasize how cool this is! The CV changes the result. If you don't CV you are too confident in an overfit model that tells you that competitor identity matters. But this is not true when CVing!

Reply:
Dang, you're right. I was so lost in the details of getting this paper out I didn't even realize this.